_includes/tutorials/console-consumer-read-specific-offsets-partition/kafka/code/tutorial-steps/dev/create-topic.sh

```diff
@@ -0,0 +1 @@
+docker exec -i broker /usr/bin/kafka-topics --create --topic example-topic --zookeeper zookeeper:2181 --replication-factor 1 --partitions 2
```

Comment:
```suggestion
docker-compose exec broker kafka-topics --create --topic example-topic --bootstrap-server broker:9092 --replication-factor 1 --partitions 2
```

/usr/bin is in the system path, so I'm not sure we need to clutter the command w/ the full path.  Also modified to use bootstrap-server and docker-compose exec instead of docker exec

Reply:
I'm in the process of converting the 101 PRs to use `docker-compose` I've already updated https://github.com/confluentinc/kafka-tutorials/pull/380 I'm doing so for this one now.

Reply:
actually just tried this and the `/usr/bin` is required. But I'm thinking of following the pattern of opening a shell on the container first

Reply:
That's interesting. What could be preventing `/usr/bin` from being on the PATH for you?

Seems to work with bare commands:
`$ docker run confluentinc/cp-kafka:5.5.0 sh -c 'echo $PATH'`
```
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
--
`$ docker run confluentinc/cp-kafka:5.5.0 sh -c 'which kafka-topics'`
```
/usr/bin/kafka-topics
```

Reply:
It works for me without the full path:

```
15:55:54 in dev/tmp/console-consumer-read-specific-offsets-partition took 2s
[I] ➜ docker-compose exec broker kafka-topics --create --topic example-topic --bootstrap-server broker:9092 --replication-factor 1 --partitions 2
Created topic example-topic.

```

Reply:
not sure, but I'm not too concerned as I've followed the pattern of having users open shell on the broker container and then use `kafka-topics...` . This follows the same path we have for users to run the console consumer and producer in these tutorials.

Reply:
Just to close the loop, I must have fat-fingered something before as it works now, but I'm going with the container shell option.  I'll need to update the other PR and the one I just merged earlier for consistency